### PR TITLE
search across contacts

### PR DESCRIPTION
**Pull request by AndrewCLu**

### CODE REVIEW

**Comment by stevenelleman** [comment]
image

What did Rachel / Tessa think about having separate sectioning for each search area (e.g. contact, bio, notes)? It definitely offers some benefits of specificity but it feels redundant to put the same contact many times on one page -- for every redundant contact there are fewer unique contacts that fit in view.

File: apps/frontend/src/pages/people/index.tsx [hunk 1]
```diff
@@ -204,14 +232,219 @@ const PeoplePage: React.FC = () => {
...
       header={
         <div className="flex flex-col">
-          <span className="text-label-primary text-xl leading-none font-bold tracking-[-0.1px] py-4">
-            {`Connections (${Object.keys(connections)?.length})`}
-          </span>
+          <div className="flex flex-row items-center justify-between">
+            <span className="text-label-primary text-xl leading-none font-bold tracking-[-0.1px] py-4">
+              {`Connections (${Object.keys(connections)?.length})`}
+            </span>
+            <div className="ml-auto">
```
Comment (stevenelleman) [inline apps/frontend/src/pages/people/index.tsx:438]: In the designs the search icon is right-aligned: 
image

Comment (stevenelleman) [reply apps/frontend/src/pages/people/index.tsx:438]: I solved the *same* styling issue here using `marginLeft: auto`: https://github.com/cursive-team/connections/pull/240/commits/324436c393bf9015e4e4ceed62133bf81416942f 

If you know a tailwind class for auto margin I'd love it.

Comment (RiverRuby) [reply apps/frontend/src/pages/people/index.tsx:438]: should be `ml-auto`

Comment (stevenelleman) [reply apps/frontend/src/pages/people/index.tsx:438]: ah makes sense, thanks

Comment (AndrewCLu) [reply apps/frontend/src/pages/people/index.tsx:438]: i've tried both of these, no luck

**Review by stevenelleman — APPROVED** [review]

**Pull request merged**